orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteStageHandler.kt

```diff
@@ -72,6 +67,14 @@ class CompleteStageHandler(
           }
         }
 
+        if (status.isFailure) {
+          val onFailureStages = stage.planOnFailureStages()
```

Comment:
@duftler As discussed, an example of where we can "dynamically" alter stages in a pipeline